Lib9c/Action/CombinationEquipment15.cs

```diff
@@ -0,0 +1,660 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Linq;
+using Bencodex.Types;
+using Lib9c.Abstractions;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume.Extensions;
+using Nekoyume.Helper;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.Mail;
+using Nekoyume.Model.Skill;
+using Nekoyume.Model.Stat;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using Nekoyume.TableData.Crystal;
+using Serilog;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    /// <summary>
+    /// Hard forked at https://github.com/planetarium/lib9c/pull/1334
```

Comment:
I don't want to discuss on this part but this comment doesn't seem to work.

Because https://github.com/planetarium/lib9c/pull/1334 was closed.

You can ignore this comment now but it will be good to discuss about this if you want. The hard-fork comments are still useful?